### ISSUE TITLE
Naming Scheme

### ISSUE THREAD

**Issue by cacsphysics** [Bryn-Mawr-Plasma-Laboratory/bmplib#3]
@dschaffner Which [naming scheme](https://www.python.org/dev/peps/pep-0008/#naming-conventions) should the bmplib package use? 
I like the _mixedCase_ and the _lower_case_with_underscores_.

This way there is consistency within the package.

**Comment by dschaffner** [comment]
I think we should make it similar to PlasmaPy and/or bapsflib. Maybe @rocco8773 can chime in here?

**Comment by rocco8773** [comment]
You're a python package, so I'd follow [PEP8 naming conventions](https://www.python.org/dev/peps/pep-0008/#naming-conventions).

**Comment by dschaffner** [comment]
Right, but I think what Carlos is asking is about which naming convention to actually use. The link you sent seems to suggest there are different variations. What is the variation you and Nick tend to use?

**Comment by rocco8773** [comment]
PEP8 naming conventions for [classes are CamelCase](https://www.python.org/dev/peps/pep-0008/#class-names), [functions are lowercase_with_underscores](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names) (same goes for variables and methods), [constants are CAPITALS_WITH_UNDERSCORES](https://www.python.org/dev/peps/pep-0008/#constants), etc.  If you look through [PEP8 it has naming specifications for each type](https://www.python.org/dev/peps/pep-0008/#id37).
